autodiff-template

```diff
@@ -2,17 +2,16 @@
 //********************************************************************
 //
 // Global defines
-// 
```

Comment:
I've set my editor to automatically strip trailing whitespace. That's what this change is. It seemed benign to me, but I'm happy to turn it off if you'd prefer that.